### PR TITLE
Add Devise

### PR DESCRIPTION
**Pull request by Whoistolu**
### In this milestone, we:
- Removed current_user method in ApplicationController.
- Install and set up devise.
    - Registers a new user.
    - User logs in with a combination of email and password.
    - Hashed passwords should be stored in the database.
    - Asks for confirmation of email.
    -  Resets password.

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
```
Comment (Dipeshtwis) [inline app/views/devise/sessions/new.html.erb]: - [x] The view should be modified to match the wireframes and their styling.

### Your design

Screenshot from 2022-02-02 15-05-58


### Given design
Screenshot from 2022-02-01 18-46-05

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Mhdez221993, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️
 
**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (Whoistolu) [reply app/views/devise/sessions/new.html.erb]: Thank you for the feedback @Dipeshtwis . This will be implemented right away.

Comment (Mhdez221993) [reply app/views/devise/sessions/new.html.erb]: login_blog
sigup_blog

Comment (Mhdez221993) [reply app/views/devise/sessions/new.html.erb]: Done! 👍

**Review by dasileker — APPROVED** [review]
## **Hi @Whoistolu,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dasileker** [comment]
## **Hi @Whoistolu,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 
### **Note**

- **Please do not merge or close till everyone working on the repo get the approved message.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**